src/views/preview/comment/comment.jsx

```diff
@@ -71,15 +112,19 @@ class Comment extends React.Component {
                                     Delete {/* TODO internationalize */}
```

Comment:
It's probably a good time to localize 'Delete' and 'Report'.

Reply:
Fixed in c19adee